.Rbuildignore

```diff
@@ -4,4 +4,5 @@
 msgpack.org.md
 rebuild.sh
 .*.tar.gz
-^local
\ No newline at end of file
+^local
+rticle
```

Comment:
I tend to stick other material inside a directory `local/`, then I need only one entry here and in `,gitignore`